hyph-utf8/tex/generic/hyph-utf8/patterns/tex/hyph-sl.tex

```diff
@@ -35,6 +42,10 @@
 %             WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
 %             FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 %             OTHER DEALINGS IN THE SOFTWARE.
+% hyphenmins:
+%     typesetting:
+%         left: 2
+%         right: 3
```

Comment:
As mentioned in [the commit message](https://github.com/hyphenation/tex-hyphen/commit/9f68d33), this reflects the `hyphenmin` values that were in [the original 8-bit file](https://github.com/hyphenation/tex-hyphen/blob/9f68d33/old/hyphen/sihyph.tex).  We must have overlooked it when creating the YAML header files.